sh.sh

```diff
@@ -0,0 +1,32 @@
+read _ _ KERNEL _ </proc/version
+
+[ -z "$HOSTNAME" ] && read HOSTNAME </etc/hostname
+
+TASKS=0
+for dir in /proc/*; do
+  case $dir in
+  	*[0-9]*) TASKS=$((TASKS + 1)) ;;
+  esac
+done
+
+set --
+
+while IFS=$'\n' read -r l; do
+	set -- $@ $l
+done </proc/meminfo
+
+TOTAL=$(($2 / 1000))
+AVAIL=$(($8 / 1000))
```

Comment:
the 2 and 8 are too magic, sorry, try something else. i guess knowing to use the first and third lines is also magaical, easier to verify at a glance.